MYabrv.bib

```diff
@@ -263,13 +273,17 @@ @String{Springer
  @String{BerlinHeidelberg = "Berlin, Heidelberg"}
  @String{London = "London, UK"}
  @String{Secaucus = "Secaucus, NJ, USA"}
+ @String{IBFI = "IBFI"}
 @String{IOS = "IOS Press"}
 @String{PrenticeHall = "Prentice-Hall Inc."}
  @String{UpperSaddleRiver = "Upper Saddle River, NJ, USA"}
 @String{Kluwer = "Kluwer Academic Publishers"}
  @String{Hingham = "Hingham, MA, USA"}
 @String{SEI = "Software Engineering Institute"}
  @String{Pittsburgh = "Pittsburgh, PA, USA"}
+ @String{VLDB = "VLDB Endowment"}
+@String{CiteSeer = "CiteSeer"}
```

Comment:
That's not a publisher, or?

Reply:
Avoid getting entries from Citeseer and any wrong data like that. As far as I know, Citeseer is not a publisher.

Reply:
Ok, I will remove my single reference from CiteSeer then. I cannot find any other source for that reference.